addons/website_forum/static/src/js/website_forum.js

```diff
@@ -501,6 +508,62 @@ publicWidget.registry.websiteForum = publicWidget.Widget.extend({
         $('.forum_intro').slideUp();
         return true;
     },
+    /**
+     * @private
+     * @param {Event} ev
+     */
+    _onFlagValidatorClick: async function (ev) {
+        ev.preventDefault();
+        const currentTarget = ev.currentTarget;
+        const result = await this._rpc({
+            model: 'forum.post',
+            method: currentTarget.dataset.action,
+            args: [parseInt(currentTarget.dataset.postId)],
+        });
+        if (result) {
+            currentTarget.parentElement.classList.toggle('d-none');
+            const flaggedButton = currentTarget.parentElement.previousElementSibling,
+                child = flaggedButton.firstElementChild,
+                countFlaggedPosts = this.el.querySelector('#count_flagged_posts'),
+                count = parseInt(countFlaggedPosts.innerText, 10) - 1;
+
+            flaggedButton.innerText = _t(' Flag');
+            flaggedButton.prepend(child);
+            if (count === 0) {
+                countFlaggedPosts.classList.add("badge-light");
+            }
+            countFlaggedPosts.innerText = count;
+        }
+    },
+    /**
+     * @private
+     * @param {Event} ev
+     */
+    _onFlagMarkAsOffensiveClick: async function (ev) {
+        ev.preventDefault();
+        const result = await this._rpc({
+            route: $(ev.currentTarget).data('action'),
+        });
+        if (result && result.error) {
+            this.call('crash_manager', 'show_warning', {
```

Comment:
And.. I actually tried your code because why not and..
Boom :boom: traceback

![image](https://user-images.githubusercontent.com/30048408/129196628-c37b63b1-1d5c-4c7e-adf0-fe669c257beb.png)

Please, retest all the flows of this PR before saying "done" after this review.

I get this task is probably a task that was done before owl, and it was working at some point, but we already had a few reviews these last few days and you should have tried such flows after rebasing a x months old PR.
That's the bare minimum when sending a PR to review:
1. Check your code is linted ok
2. Check your code works

Note that call `crash_manager` shoudln't have been written in the first place, we always have called displayNotification